src/tests/Conversion.test.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
As much as possible, kindly avoid disabling linter checks in-line. Linters represent coding standards you share with your team and most of the time enforce adhering to best coding practices.